app/services/auth/google_service.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Auth
+  class GoogleService < BaseService
+    BASE_SCOPE = %w[profile email openid].freeze
+
+    def initialize
+      @client_id = Google::Auth::ClientId.new(ENV['GOOGLE_AUTH_CLIENT_ID'], ENV['GOOGLE_AUTH_CLIENT_SECRET'])
+
+      super
+    end
+
+    def authorize_url(request)
+      authorizer = Google::Auth::WebUserAuthorizer.new(
+        client_id,
+        BASE_SCOPE,
+        nil,
```

Comment:
maybe a comment to name this parameter? 😅

Reply:
its a nil parameter, no one knows about it 😂